src/components/Overlay/Overlay.scss

```diff
@@ -27,3 +27,7 @@
 .ms-Overlay--dark {
   background-color: $ms-color-blackTranslucent40;
 }
+
+.ms-noScroll {
```

Comment:
Thanks @mikewheaton.  I wasn't aware of the -u- prefix as a utility class.  I'll rename and recommit now.